packages/react-flex/src/components/Flex/useFlexStyles.ts

```diff
@@ -16,11 +25,22 @@ export const useFlexStyles = (state: FlexState): FlexState => {
   const styles = useStyles();
   state.className = mergeClasses(styles.root, state.className);
 
-  // TODO: Hook up slot styles, for example:
-  // const exampleSlotClassName = useExampleSlotStyles(state);
-  // if (state.exampleSlot) {
-  //   state.exampleSlot.className = mergeClasses(exampleSlotClassName, state.exampleSlot.className);
-  // }
+  const direction = state.direction ?? 'row';
+  const horizontalAlign = state.horizontalAlign ?? 'normal';
+  const verticalAlign = state.verticalAlign ?? 'normal';
+
+  state.style = {
+    ...state.style,
+    display: state.inline ? 'inline-flex' : 'flex',
+    flexDirection: direction,
+    flexWrap: state.wrap ? 'wrap' : 'nowrap',
+    justifyContent: direction.startsWith('row') ? horizontalAlign : verticalAlign,
```

Comment:
I'm sorry for being that guy, but `startsWith` is not supported by IE11 and we still don't have a solid stance to discontinue support for it

Reply:
That's a good point. Unfortunately the solution is also not supported in IE11 because of CSS vars. Lets follow up on how we're standing with IE11 support.

Reply:
Let's not break things where we have the option to not break :)

Reply:
Definitely! I'll change `.startsWith` but let us see about IE11 supporting regardless.